pypeit/core/gui/identify.py

```diff
@@ -701,17 +715,27 @@ def store_solution(self, final_fit, binspec, rmstol=0.15,
                                                              (self.specdata.size - 1))
                 wvutils.write_template(wavelengths, self.specdata, binspec,
                                          './', 'wvarxiv.fits')
-                msgs.info("\nYour arxiv solution has been written to wvarxiv.fits")
-                #msgs.info("\nYour wavelength solution has been stored here:" + msgs.newline() +
-                #          os.path.join(master_dir, outroot) + msgs.newline() + msgs.newline() +
-                #          "If you would like to move this to the PypeIt database, please move this file into the directory:" +
-                #          msgs.newline() + templates.outpath + msgs.newline() + msgs.newline() +
-                #          "Please consider sending your solution to the PypeIt team!" + msgs.newline())
-                #
+
+                # Also copy the file to the cache for direct use
+                date_str = datetime.now().strftime("%Y%m%dT%H%M")
+                cachename = f"manual_{self.specname}_{date_str}.fits"
+                data.write_file_to_cache("wvarxiv.fits",
+                                         cachename,
+                                         "arc_lines/reid_arxiv")
+
+                msgs.info("Your arxiv solution has been written to ./wvarxiv.fits\n")
+                msgs.info(f"Your arxiv solution has been cached.{msgs.newline()}"
+                          f"Use 'reid_arxiv = {cachename}' and{msgs.newline()}"
+                          f"'method = full_template' in your PypeIt{msgs.newline()}"
+                          "Reduction File to utilize this wavelength solution.")
```

Comment:
Should we add the explicit lines of code the user needs to copy paste into their reduction file?

```
[calibrations]
  [[wavelengths]]
    reid_arxiv = {cachename}
    method = full_template
```

Reply:
Added.